readme.md

```diff
@@ -278,7 +278,14 @@ to start with “kernel documentation”, which is dense).
     first edition of a workshop to get started with XDP.
 -   [XDP for the Rest of Us](https://www.netdevconf.org/2.2/session.html?gospodarek-xdp-workshop)
     from the same authors, at Netdev 2.2: second edition, with new contents.
--   [Notes on bpf](https://blogs.oracle.com/linux/notes-on-bpf-1) Some wonderful tutorial or notes about (e)bpf from oracle.
+-   Generic Documentation and Presentions from oracle's Greg Marsden
+    -	[eBPF Program type](https://blogs.oracle.com/linux/notes-on-bpf-1) Some notes aboye ebpf program_type.
+    -	[BPF helper functions](https://blogs.oracle.com/linux/notes-on-bpf-2) BPF helper functions
```

Comment:
Again, do not repeat the title for the description. Find something else, or just remove the description entirely if the title is descriptive enough.

“A review of the kernel functions that can be called from within eBPF programs”?